espnet/bin/asr_train.py

```diff
@@ -164,6 +169,10 @@ def main(args):
                         help='Gradient norm threshold to clip')
     parser.add_argument('--num-save-attention', default=3, type=int,
                         help='Number of samples of attention to be saved')
+
+    from espnet.nets.pytorch_backend.e2e_transformer import add_transformer_arguments
```

Comment:
OK I think it is possible like this way
```python
def main(cmd_args):
  args = parse.parse_args(cmd_args)
  if args.ntype == "transformer":
    args = add_transformer_arguments(parser).parse_args(cmd_args)
```

Reply:
I was wrong. Argument parser does not accept unregistred options like `--transformer-init`. Maybe we can avoid name conflictions by adding prefix `--transformer-xxx` for something not needed in the default RNN model.

Reply:
Or we can repeat every options needed in the model like fairseq. I think it can be more robust to unexpected behaviours.
- LSTM modfel https://github.com/pytorch/fairseq/blob/d259ffa98d90fb3100ffdab70c4db65c06b6b552/fairseq/models/lstm.py#L26
- Transformer model https://github.com/pytorch/fairseq/blob/d259ffa98d90fb3100ffdab70c4db65c06b6b552/fairseq/models/transformer.py#L50